guides/common/modules/proc_switching-scc-accounts.adoc

```diff
@@ -5,7 +5,7 @@ You can switch your SCC account by changing the SCC credentials saved on {Projec
 
 [WARNING]
 ====
-The SCC Manager plugin does not support multiple SCC accounts.
+The SCC Manager plugin does not support multiple SCC accounts within one organization but for the purpose of switching SCC accounts.
```

Comment:
```suggestion
The SCC Manager plugin does not support multiple SCC accounts within one organization, except for when switching SCC accounts.
```
I'm not entirely sure about this, but it would be nice to replace 'but' with something that's more in line with simple English.

Reply:
I like this suggestion -> applied.